### PR TITLE
ci/rework-include-ts

### PR DESCRIPTION
**Pull request by Charles-Schleich**
The CI/CD pipeline should include releasing 3 things 
- `Zenoh-ts` packaged and released on npm (github) 
- `zenoh-plugin-remote-api` released on github
- Documentation publishing 
All of these things should happen on a push to a version tag
This requires all `Cargo.toml`'s and `package.json`'s to be bumped to the same version number. 

The respective workflows for plugin, docs and plugin will handle publishing.

Debate: Should we allow releasing the library or the plugin manually as well ?

### CODE REVIEW

File: .github/workflows/release-plugin.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+name: Release Plugin
 
 permissions:
   contents: write
 
 on:
   push:
     tags:
-      - v[0-9]+.*
+      - 'v[0-9]+.*'
```
Comment (diogomatsubara) [inline .github/workflows/release-plugin.yml:10]: without the v

File: .github/workflows/release-ts.yml [hunk 2]
```diff
@@ -1,24 +1,9 @@
...
-        type: string
-      zenoh_version:
-        description: 'Zenoh version'
-        required: true
-        type: string
-        default: '1.0.0-alpha.6'
-  push: 
+  push:
+    tags:
+      - v[0-9]+.*
```
Comment (diogomatsubara) [inline .github/workflows/release-ts.yml:6]: without the v and quoted

File: .github/workflows/release-plugin.yml [hunk 3]
```diff
@@ -1,12 +1,12 @@
-name: Release
+name: Release Plugin
```
Comment (diogomatsubara) [inline .github/workflows/release-plugin.yml:2]: I think this whole workflow can be deleted. We have in the `release.yml` a job to publish the artifacts created during the build to github and release it as a Github release.

**Review by diogomatsubara — CHANGES_REQUESTED** [review]

File: .github/workflows/release-plugin.yml (hunk 3 above)
Comment (Charles-Schleich) [reply .github/workflows/release-plugin.yml:2]: Removed the workflow, i was under the impression the workflow was doing the releasing.

**Comment by Charles-Schleich** [comment]
@gabrik ive opened an issue in gitlab for the Eclipse Foundation to create a bot on NPM, 
once they create that bot i will integrate publishing to npm in the CI/CD (Should just require an update to the .npmrc)  
 https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/5279

**Review by diogomatsubara — APPROVED** [review]

**Pull request merged**